derive/src/rpc_attr.rs

```diff
@@ -40,6 +41,7 @@ const RAW_PARAMS_META_WORD: &str = "raw_params";
 const SUBSCRIBE_META_WORD: &str = "subscribe";
 const UNSUBSCRIBE_META_WORD: &str = "unsubscribe";
 const RETURNS_META_WORD: &str = "returns";
+const NOTIFICATION_ATTR_NAME: &str = "notification";
```

Comment:
Is this still in use? Seems not needed in the example.

Reply:
Whoops, I forgot to remove the attribute from the code while making the changes. Good catch! 👍